panel_client/keyboard_panel.py

```diff
@@ -0,0 +1,96 @@
+"""Demonstration/debugging panel. Uses the shell for I/O."""
```

Comment:
This file seems kinda redundant with `perfect_panel.py`?

Reply:
CL spam. perfect_panel has been removed.